app/src/shared/utils/getClientConfig.js

```diff
@@ -1,18 +1,20 @@
 import { getToken } from '$shared/utils/sessionToken'
 
 export default function getClientConfig(options = {}) {
-    const sessionToken = getToken()
-
-    const auth = sessionToken == null ? {} : {
-        sessionToken,
-    }
-
-    return Object.assign({
-        auth,
+    const config = Object.assign({
         autoConnect: true,
         autoDisconnect: false,
         restUrl: process.env.STREAMR_API_URL,
         url: process.env.STREAMR_WS_URL,
         verifySignatures: 'never',
     }, options)
+
+    const sessionToken = getToken()
+    if (sessionToken) {
+        config.auth = {
```

Comment:
I noticed that providing `auth: {}` will not work and websocket connection will be in a weird state. Fixed this by providing `auth` only if we have a `sessionToken`.

Reply:
I wonder what are the cases for this? At least when viewing a product's stream when not logged in. The difference I observed was that the progress bar is spinning indefinitely when not logged in but it's not because of this change (same happens in `development`).

Reply:
That and also when viewing a data union stats on Marketplace (not logged in) or in DU details (logged in).